tests/projects/SelfContained_Trimming_Test/Program.fs

```diff
@@ -11,6 +12,12 @@ let report_failure (s : string) =
     failures := !failures @ [s]
     ()
 
+let check s e r = 
```

Comment:
Would appreciate some clearer variable names here

Reply:
this does not matter, we don't run those, only trimming, we can change everything to "potato" and it won't matter.

Reply:
@psfinaki in this case, I am with @vzarytovskii these particular functions, are replicated across a ton of test cases in the cambridge suite.  I was really just looking for something to verify that trim didn't change the behavior.